src/Parser/Warrior/Protection/Modules/Talents/HeavyRepercussions.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import calculateEffectiveDamage from 'Parser/Core/calculateEffectiveDamage';
+import { formatNumber, formatPercentage } from 'common/format';
+
+const HEAVY_REPERCUSSIONS_SHIELD_BLOCK_EXTEND_MS = 1000;
+const HEAVY_REPERCUSSIONS_SHIELD_SLAM_DAMAGE_BUFF = 0.3;
+
+class HeavyRepercussions extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    enemies: Enemies,
+  };
+
+  sbExtended = 0;
+  ssBuffRefreshed = 0;
+  nextShieldSlamBuffed = false;
```

Comment:
I don't play prot, but reading the Heavy Repercussions tooltip it implies that all Shield Slams are buffed during Shield Block, not just the first one. Are you sure this is how it works?

Reply:
You're right, it's buffing all SS's during SB. Not sure why I thought it's per SB cast.